README.md

```diff
@@ -47,6 +48,25 @@ end
 Please see the documentation for [OAuth2.Serializer](https://hexdocs.pm/oauth2/OAuth2.Serializer.html)
 for more details.
 
+## Configure a http client
+
+The http client library used is [tesla](https://github.com/elixir-tesla/tesla), the default adapter is
+httpc. But you probably want to use some better adapter.
+You can configure it like so:
+
+```elixir
+config :tesla, adapter: Tesla.Adapter.Hackney
+```
```

Comment:
We should add an example on how to change the adapter only for `:oauth2` component instead of globally. Something like:

```elixir
config :oauth2, adapter: Tesla.Adapter.Hackney
```

Or however it suppose to be done.

Reply:
You are right they have that option, missed that. Thought they only supported mix config